.github/workflows/ci.yml

```diff
@@ -475,11 +473,11 @@ jobs:
         uses: bufbuild/buf-breaking-action@a074e988ee34efcd4927079e79c611f428354c01 # v1
         with:
           input: apis
-          against: "https://github.com/${GITHUB_REPOSITORY}.git#branch=master,subdir=apis"
+          against: "https://github.com/${GITHUB_REPOSITORY}.git#branch=${GITHUB_REF_NAME},subdir=apis"
```

Comment:
Hrm, I guess we want `master` specifically if it's `ref/heads/master` or a PR, but the branch name if it's a release branch. 🤔